e2e/cypress/tests/integration/enterprise/cloud/billing/pricing_modal_spec.js

```diff
@@ -63,6 +69,104 @@ describe('Pricing modal', () => {
         cy.get('#UpgradeButton').should('not.exist');
     });
 
+    // it('should not ping admin twice before cool off period', () => {
```

Comment:
@marianunez temporarily commented out these tests until we have a way to open the pricing modal for non-admin users